kfs-ld/src/main/java/edu/arizona/kfs/module/ld/batch/service/impl/ScrubberValidatorImpl.java

```diff
@@ -87,30 +87,32 @@
             errors.add(err);
         }
 
-        err = validateGTEAccount(laborScrubbedEntry);
-        if (err != null) {
-            errors.add(err);
+        // Per v6 spec, labor entries are not to be GTE validated. Further, to keep with v3 behavior
+        // and work done under KITT-2330, Dont go through GTE validation if other errors exist -- this
+        // helps ensure reports appear as they have in the past
+        if (!laborIndicator && (errors == null || errors.isEmpty())) {
+            err = validateGTEAccount(laborScrubbedEntry);
+            if (err != null) {
+                errors.add(err);
+            }
         }
 
         return errors;
     }
 
     protected Message validateGTEAccount(LaborOriginEntry originEntry) {
         LOG.debug("validateGTEAccount() started");
-        originEntry.refreshReferenceObject("account");
```

Comment:
Removing the refreshReferenceObject should make an improvement in performance since it was just for validation reasons...